roles/restore/tasks/postgres.yml

```diff
@@ -87,11 +87,27 @@
     namespace: "{{ backup_pvc_namespace }}"
     pod: "{{ ansible_operator_meta.name }}-db-management"
     command: |
-      bash -c """
+      bash -c "
+      function end_keepalive {
+        rc=$?
+        rm -f \"$1\"
+        kill $(cat /proc/$2/task/$2/children 2>/dev/null) 2>/dev/null || true
+        wait $2 || true
+        exit $rc
+      }
+      keepalive_file=\"$(mktemp)\"
+      while [[ -f \"$keepalive_file\" ]]; do
+        echo 'Migrating data from old database...'
+        sleep 60
+      done &
+      keepalive_pid=$!
+      trap 'end_keepalive \"$keepalive_file\" \"$keepalive_pid\"' EXIT SIGINT SIGTERM
+      echo keepalive_pid: $keepalive_pid
       set -e -o pipefail
       cat {{ backup_dir }}/tower.db | PGPASSWORD='{{ awx_postgres_pass }}' {{ pg_restore }}
```

Comment:
@rooftopcellist 
Not tested yet, but is it okay to use `{{ awx_postgres_pass }}` here? It might include `'` or `"` and is still problematic imo.

Reply:
And `"` in password is also still problematic since the `awx_postgres_pass` is also used to embed password into `credentials.py`: https://github.com/ansible/awx-operator/blob/4c5429190c9342a25b78af26b81366d218dd4df2/roles/installer/templates/settings/credentials.py.j2

Reply:
@kurokobo 

I created a separate issue to track this:
* https://github.com/ansible/awx-operator/issues/1545

The changes in this PR and in https://github.com/ansible/awx-operator/pull/1540 were fairly straight-forward, so I made them.  But fixing the other two spots might not be so simple, so I deferred them to future PR's.  We haven't gotten any bug reports from users yet on this, it's just something that @relrod and I noticed in the code the other day, so I am hesitant to make changes that would add more tasks to the code (and slow down the loop) if no one is actually running into this issue.